testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/OidcUserInfoClaimToRoleMapperTest.java

```diff
@@ -0,0 +1,168 @@
+package org.keycloak.testsuite.broker;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.keycloak.admin.client.resource.IdentityProviderResource;
+import org.keycloak.broker.oidc.mappers.ClaimToRoleMapper;
+import org.keycloak.broker.provider.ConfigConstants;
+import org.keycloak.models.IdentityProviderMapperModel;
+import org.keycloak.models.IdentityProviderMapperSyncMode;
+import org.keycloak.representations.idm.IdentityProviderMapperRepresentation;
+import org.keycloak.representations.idm.IdentityProviderRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+
+import java.util.List;
+
+import static org.keycloak.models.IdentityProviderMapperSyncMode.FORCE;
+import static org.keycloak.models.IdentityProviderMapperSyncMode.LEGACY;
+
+/**
+ * @author <a href="mailto:external.martin.idel@bosch.io">Martin Idel</a>
+ */
+public class OidcUserInfoClaimToRoleMapperTest extends AbstractRoleMapperTest {
```

Comment:
Is it possible to remove some @Test methods from this class? It looks that this class is just copy/pasted from the existing class OidcClaimToRoleMapperTest and it adds all the tests, which were added by the original class. However we don't need 8 new test methods for fixing this single bug? I hope that 1 or 2 test methods should be sufficient?

I mean tests are good, but too much tests adds overhead and additional time to run the code + additional maintenance price etc :-)